### PR TITLE
Implement GET /graph endpoint

### PR DESCRIPTION
**Pull request by SourenP**

### CODE REVIEW

**Comment by SourenP** [comment]
but I couldn't figure out how to get the api doc to display on the homepage

**Comment by stevenzeiler** [comment]
Try `npm run apidoc:build`

**Comment by stevenzeiler** [comment]
Then check in those files and push to this branch please.

**Comment by stevenzeiler** [comment]
:+1: LGTM

File: config/routes.js [hunk 1]
```diff
@@ -31,4 +31,6 @@ module.exports.routes = {
   * for configuration options and examples.                                  *
   *                                                                          *
   ***************************************************************************/
+
+  'GET /graph': 'GraphController'
```
Comment (stevenzeiler) [inline config/routes.js]: Can you please explicitly specify the controller and action like here? https://github.com/ripple/validator-registry-api/blob/staging/config/routes.js

**Pull request merged**